src/util/sample.h

```diff
@@ -370,6 +370,17 @@ class SampleUtil {
             mixxx::audio::ChannelCount numChannels,
             int sourceChannel = 0);
 
+    // Copies and strips interleaved stereo sample data in pSrc with
+    // down to multi-channel samples into pDest. Samples will be written at the
+    // channel pointed by channelOffset. Samples from all other channels will be
+    // ignored. pDst must contain (numFrames * numChannels) samples (numFrames *
+    // 2) samples will be written into pDest
+    static void copyStereoToMulti(CSAMPLE* pDest,
```

Comment:
In adidition an example in the comment will be also helpfull. 

Does the term "insert" fit here? 

The layout is now like: 
1L1R2L2R1L1R2L2R1L1R2L2R1L1R2L2R
right? 

Does 
1L1R1L1R1L1R1L1R2L2R2L2R2L2R2L2R
Have any benefit? 

Do we have better name these layouts so we can use consistenly throut the code base?

Reply:
> Does the term "insert" fit here?

Sounds much better indeed. I have added an example as well.

> Does
1L1R1L1R1L1R1L1R2L2R2L2R2L2R2L2R
Have any benefit?

AFAIS, I mostly see issues and limitations. The main one being the inability to read frame sequentially and requiring to consume the entire buffer in order to reconstruct a single frame. With the current layout, you do not have to take into consideration the entire buffer size.
The only advantage of using the following layout is to drop the need of `SampleUtil::copyOneStereoFromMulti`, but since this helper is trivial, it only takes couple of ns to extract a single stereo track after vectorization.